src/Access/AccessEntityIO.cpp

```diff
@@ -85,6 +85,16 @@ String serializeAccessEntity(const IAccessEntity & entity)
     WriteBufferFromOwnString buf;
     for (const ASTPtr & query : queries)
     {
+        if (const User * user = typeid_cast<const User *>(&entity))
+        {
+            if (user->auth_data.getSalt().compare("") != 0)
+            {
+                std::string strSalt = " '";
```

Comment:
Why quotes?

Reply:
Hi Alexey, 
it was a wrong code base. I will close this and create new PR. sorry about this.